integration/spark/app/src/test/resources/log4j.properties

```diff
@@ -7,3 +7,6 @@ log4j.appender.console.layout.ConversionPattern=%d{yy/MM/dd HH:mm:ss} %p %c{1}:
 
 # set the log level for the openlineage spark library
 log4j.logger.io.openlineage=DEBUG
+log4j.logger.io.openlineage.spark.shaded=WARN
```

Comment:
this gets rid of http library debugs which makes huge diffs to logs & output produced in circle ci